src/libNode/MicroBlockPreProcessing.cpp

```diff
@@ -485,9 +485,20 @@ void Node::UpdateProcessedTransactions() {
     m_createdTxns = move(t_createdTxns);
```

Comment:
Looks like `UpdateProcessedTransactions` will be triggered only after receiving FinalBlock. `ProcessFinalBlockCore`->`LoadUnavailableMicroBlockHashes`->`UpdateProcessedTransactions`